Formula/gerbil-scheme.rb

```diff
@@ -0,0 +1,35 @@
+class GerbilScheme < Formula
+  desc "Opinionated dialect of Scheme designed for Systems Programming"
+  homepage "https://cons.io"
+  url "https://github.com/vyzo/gerbil/archive/v0.14.tar.gz"
+  sha256 "8525877960ba7a6122d6dfd2bae34ed1437d5231f9e31de97bbe6106c4dfa03a"
+
+  depends_on "gambit-scheme"
+  depends_on "leveldb"
+  depends_on "libyaml"
+  depends_on "lmdb"
+  depends_on "openssl"
+
+  def install
+    cd "src" do
+      ENV.append_path "PATH", "#{Formula["gambit-scheme"].opt_prefix}/current/bin"
+      ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :sierra
+
+      inreplace "std/build-features.ss" do |s|
+        s.gsub! "(enable leveldb #f)", "(enable leveldb #t)"
+        s.gsub! "(enable libxml #f)", "(enable libxml #t)"
+        s.gsub! "(enable libyaml #f)", "(enable libyaml #t)"
+        s.gsub! "(enable lmdb #f)", "(enable lmdb #t)"
+      end
+
+      system "./build.sh"
+    end
+
+    libexec.install "bin", "lib", "doc"
```

Comment:
Should be installed in `prefix`, not `libexec`. Or at the very least, symlink executables from `libexec/bin` into `bin`, so they are in the user's PATH

Reply:
It was set to prefix initially. However, due to the fact that lib needs non-binary library files, it was recommended to move to libexec to avoid it failing that test.

Reply:
Then you still need to symlink your binaries in `bin`, so they are in the user's `PATH`

Reply:
The binaries assume the libs will be at the same top level in the layout. A symlink for the gxi bin for example, will assume, that the libs are in the peer lib/ directory. But since we can't install the required files into the peer lib, due to their non-binary nature, it will fail.

Reply:
Links have been added and tests pass.